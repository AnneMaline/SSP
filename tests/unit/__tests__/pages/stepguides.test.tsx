import { render } from "@testing-library/react";
import StepGuidesPage from "../../../../src/app/Step-guides/page";

describe("Step-guide Page", () => {
  it("renders correctly", () => {
    const { container } = render(<StepGuidesPage />);
    expect(container).toMatchSnapshot();
  });
});
