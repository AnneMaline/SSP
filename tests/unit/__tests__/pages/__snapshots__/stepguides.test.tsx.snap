// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Step-guide Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      Step-guides
    </h1>
    <div
      class="bg-gray-100"
    >
      <h1
        class="text-2xl font-bold text-center my-6"
      >
        Step-by-step-guides
      </h1>
      <div
        class="grid grid-cols-1 md:grid-cols-3 gap-6 p-4"
      >
        <div
          class="bg-white shadow-md rounded-lg p-6 border border-gray-200"
        >
          <h2
            class="text-lg font-bold mb-2"
          >
            React Documentation
          </h2>
          <p
            class="text-gray-700 mb-4"
          >
            Learn more about React and its features on the official documentation.
          </p>
          <a
            class="text-blue-500 hover:underline"
            data-link-title="React Documentation"
            href="https://reactjs.org"
            rel="noopener noreferrer"
            target="_blank"
          >
            Learn More
          </a>
        </div>
        <div
          class="bg-white shadow-md rounded-lg p-6 border border-gray-200"
        >
          <h2
            class="text-lg font-bold mb-2"
          >
            Next.js
          </h2>
          <p
            class="text-gray-700 mb-4"
          >
            Next.js provides the best developer experience for building React applications.
          </p>
          <a
            class="text-blue-500 hover:underline"
            data-link-title="Next.js"
            href="https://nextjs.org"
            rel="noopener noreferrer"
            target="_blank"
          >
            Learn More
          </a>
        </div>
        <div
          class="bg-white shadow-md rounded-lg p-6 border border-gray-200"
        >
          <h2
            class="text-lg font-bold mb-2"
          >
            Playwright
          </h2>
          <p
            class="text-gray-700 mb-4"
          >
            Automate tests for your web applications with Playwright.
          </p>
          <a
            class="text-blue-500 hover:underline"
            data-link-title="Playwright"
            href="https://playwright.dev"
            rel="noopener noreferrer"
            target="_blank"
          >
            Learn More
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
